django/http/response.py

```diff
@@ -408,6 +409,11 @@ def content(self, value):
             content = self.make_bytes(value)
         # Create a list of properly encoded bytestrings to support write().
         self._container = [content]
+        self.__dict__.pop("text", None)
+
+    @cached_property
+    def text(self):
+        return self.content.decode(self.charset or "utf-8")
```

Comment:
Should we handle exceptions in this method? (Specifically `UnicodeDecodeError`)
Shall we try to use am error handling strategy like `replace`, `backslashreplace` or `surrogateescape`? Or shall we let it surface to the caller?

@sarahboyce @adamchainz What do you think?

Reply:
I have no strong opinion :thinking:

Reply:
Let it raise. An exception here would also mean an error on the client side (browser or whatever).